test/reporter.test.js

```diff
@@ -1,14 +1,57 @@
-var path = require('path')
-var fs = require('fs')
-
-var promisify = require('../promisify')
 var reporter = require('../reporter')
-var reports = require('./snapshots/reports')
+
+function normalizeNewlines (string) {
+  // use local copy of Jest newline normalization function
+  // until Jest doens't apply normalization on comprasion
+  return string.replace(/\r\n|\r/g, '\n')
```

Comment:
hmm… didn't find `\v\n` in produced jest output, but i'm on macos.
need to dig on it.